From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

msmbaldwin committed:
Revised titles and added UK docs

manish-shukla01 committed:
changed file name to lower case for window software file

barclayn committed:
updating the steps to apply to the new portal

manish-shukla01 committed:
changed for file name to be lower case

tfosmark committed:
Merge pull request #27650 from manish-shukla01/1113prchanges

edited based on PR reviewer comments

barclayn committed:
adding images

barclayn committed:
adding powershell to display a secret stored in keyvault as plain text

barclayn committed:
fixing indentation for one of the images

msmbaldwin committed:
Final TOC revisions

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into load-balancer-ha-ports-overview

Ja-Dunn committed:
fixes

msmbaldwin committed:
Final TOC revisions

itechedit committed:
edit pass: sap-high-availability-infrastructure-wsfc-file-share

msmbaldwin committed:
Updated dates

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1023

itechedit committed:
Update sap-high-availability-infrastructure-wsfc-file-share.md

itechedit committed:
Update sap-high-availability-infrastructure-wsfc-file-share.md

tdykstra committed:
Cory feedback

moderakh committed:
1.15.0 release

1.15.0 release @mimig1 @kirankumarkolli

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

eamonoreilly committed:
Adding watcher tutorial

itechedit committed:
edit pass: sap-high-availability-installation-wsfc-file-share

itechedit committed:
Update sap-high-availability-installation-wsfc-file-share.md

itechedit committed:
Update sap-higher-availability-architecture-scenarios.md

Commit:
task 656: Rewrite text explaining query string structure

barclayn committed:
making modifications to the cli article

barclayn committed:
changing intro pagraph

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1023

tamram committed:
fixes per feedback

v-anpasi committed:
Bringing even with master.

vikramdesai01 committed:
address minor review feedback

vikramdesai01 committed:
address minor review feedback

ShawnJackson committed:
edit pass: api-management-howto-use-managed-service-identity

eamonoreilly committed:
Fixed indent

eamonoreilly committed:
left align variables image

eamonoreilly committed:
Fixed bullet item

eamonoreilly committed:
Fixed heading

MatkovIvan committed:
Update version of the SDK

eamonoreilly committed:
Adding watcher task to TOC

Commit:
Corrected the not supported IP restriction

Commit:
Merge branch 'master' of https://github.com/johndeu/azure-docs-pr; branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Corrected the not supported IP restriction

Commit:
more

GitHubber17 committed:
Merge pull request #27877 from vikramdesai01/address-reveiewfeedback-release-reserve-instance

Address review feedback release reserve instance

Commit:
tweak

MicrosoftGuyJFlo committed:
New error strings for on-premises integration page from PG

MicrosoftGuyJFlo committed:
New error strings for on-premises integration page from PG

msmbaldwin committed:
Final revisions

adegeo committed:
Retire old portal content; fix links; redirects

adegeo committed:
Fix link

Commit:
update bash tooling

adegeo committed:
Fix links for remote desktop

andynava committed:
No need for the reader to have GitHub account

The way the quickstart is worded, it does not require the reader to use their own GitHub account, so I think this requirement should be removed from the documentation.

Another option would be to add wording that references the reader's GitHub account...

The _deployment-source-url_ parameter is a sample repository in GitHub that contains a "Hello World" HTTP triggered function. You can replace this URL with your own GitHub repository that contains your function code.

Commit:
update owner of TF example

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into api-management-howto-use-managed-service-identity

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1023

eamonoreilly committed:
Fixed major issues from feedback.

ggailey777 committed:
Fixed the Azure CLI version

ggailey777 committed:
Update functions-create-function-linux-custom-image.md

ShawnJackson committed:
edit pass: api-management-howto-use-managed-service-identity

anrothMSFT committed:
fixed typo

anrothMSFT committed:
removing content for emotion for video (deprecated)

anrothMSFT committed:
remove from TOC emotion for video (deprecated)

Commit:
fixing merge conflicts

rwike77 committed:
Freshness update

itechedit committed:
Update sap-high-availability-installation-wsfc-file-share.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into two-active-directory-passwords

tfitzmac committed:
remove deploy samples

ktoliver committed:
Merge pull request #27952 from anrothMSFT/patch-9

removing content for emotion for video (deprecated)

edmacauley committed:
new screenshot for release with full browser and cropping older screenshot

Commit:
perf to bandwidth

ktoliver committed:
Merge pull request #27953 from anrothMSFT/patch-10

remove from TOC emotion for video (deprecated)

anandyadavMSFT committed:
clarifying version number required for feature

anandyadavMSFT committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into updatessldoc

MarkMcGeeAtAquent committed:
small fixes

ktoliver committed:
Merge pull request #27950 from anrothMSFT/patch-7

fixed typo in url

ktoliver committed:
Merge pull request #27937 from jluk/master

update bash tooling

ktoliver committed:
Merge pull request #27954 from rwike77/patch-27

Freshness update

rjagiewich committed:
Merge pull request #27933 from Thraka/csretire

Retire old portal content; fix links; redirects

itechedit committed:
Update sap-higher-availability-architecture-scenarios.md

Ja-Dunn committed:
Merge pull request #27949 from ShawnJackson/api-management-howto-use-managed-service-identity

edit pass: api-management-howto-use-managed-service-identity

kgremban committed:
IoT Edge - tutorial cleanup

mahesh-unnikrishnan committed:
Update active-directory-ds-admin-guide-configure-secure-ldap.md

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov15-tutorialreview

mahesh-unnikrishnan committed:
Update active-directory-ds-admin-guide-configure-secure-ldap.md

rjagiewich committed:
Merge pull request #27956 from tfitzmac/1115samples

remove deploy samples

mahesh-unnikrishnan committed:
Update active-directory-ds-deploy-azure-app-proxy.md

anandyadavMSFT committed:
correcting version number

kgremban committed:
Added note to ASA tutorial about regional availability

mahesh-unnikrishnan committed:
Update active-directory-ds-deploy-azure-app-proxy.md

Commit:
edit pass: active-directory-passwords-faq

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov15-tutorialreview

rjagiewich committed:
Merge pull request #27958 from edmacauley/localworking

new screenshot for release with full browser and cropping older scree…

v-thepet committed:
fix links

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1023

ShawnJackson committed:
Update sap-high-availability-infrastructure-wsfc-file-share.md

PRMerger19 committed:
Merge pull request #27657 from HemantMahawar/pscloudshell-version

Updated AzurePS and AzCli versions

tfitzmac committed:
clarified messaging types

tamram committed:
fixing links

ShawnJackson committed:
Merge pull request #27342 from itechedit/sap-higher-availability-architecture-scenarios

edit pass: sap-higher-availability-architecture-scenarios

ShawnJackson committed:
Merge pull request #27821 from itechedit/sap-high-availability-installation-wsfc-file-share

edit pass: sap-high-availability-installation-wsfc-file-share

ktoliver committed:
Merge pull request #27967 from kgremban/nov15-ASAregions

Added note to ASA tutorial about regional availability

ktoliver committed:
Merge pull request #27965 from ggailey777/connect

Fixed the Azure CLI version

ktoliver committed:
Merge pull request #27963 from mahesh-unnikrishnan/patch-17

Update active-directory-ds-deploy-azure-app-proxy.md

tynevi committed:
Merge pull request #27494 from msmbaldwin/blueprintdocs

Migrating Architecture Blueprint docs

ktoliver committed:
Merge pull request #27962 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-admin-guide-configure-secure-ldap.md

Commit:
Kumud and Jons changes

ShawnJackson committed:
Merge pull request #27777 from itechedit/sap-high-availability-infrastructure-wsfc-file-share

edit pass: sap-high-availability-infrastructure-wsfc-file-share

Commit:
changes for v6.0CU2

kgremban committed:
Added screenshots to quickstart/tutorial1 windows

ShawnJackson committed:
Update active-directory-passwords-faq.md

cychua committed:
Fix spelling error

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov15-tutorialreview

cychua committed:
Update logic

ktoliver committed:
Merge pull request #27969 from tfitzmac/1115message

clarified messaging types

ktoliver committed:
Merge pull request #27968 from v-thepet/newfix

production bug: fix links

spelluru committed:
auth key

spelluru committed:
expression

ktoliver committed:
Merge pull request #26282 from tamram/tamram-1023

moving scalability targets under VMs

Commit:
added notes about ADL being unsupported

gsacavdm committed:
Fixing typo

Commit:
Hoping this clears the error

v-anpasi committed:
Bringing even with master.

Commit:
find version changed

JYOTHIRMAISURI committed:
few updates to itsmc articles

mayanknayar committed:
Date refresh for ASR articles

Updating freshness for ASR articles

cephalin committed:
MVC18 bug

Commit:
Updates.

Commit:
Deprecated RM CLI V1

Commit:
Fixes.

Commit:
Fixes

Commit:
Removed file-

Commit:
now it wants real time hyphenated.

rjagiewich committed:
Merge pull request #27966 from kgremban/nov15-tutorialreview

IoT Edge - tutorial cleanup

tfosmark committed:
Merge pull request #27996 from cephalin/usability

MVC18 bugs

gsacavdm committed:
Fixing typo

tfosmark committed:
Merge pull request #27815 from moderakh/patch-12

1.15.0 release

tfosmark committed:
Merge pull request #27892 from johndeu/master

Modified the IP restriction warning note

tfosmark committed:
Merge pull request #27961 from anandyadavMSFT/updatessldoc

Updatessldoc

tfosmark committed:
Merge pull request #27980 from spelluru/authkey1115

auth key

tfosmark committed:
Merge pull request #27982 from spelluru/expression1115

expression

tfosmark committed:
Merge pull request #27985 from hning86/master

added notes about ADL being unsupported

tfosmark committed:
Merge pull request #27991 from mayanknayar/master

Date refresh for ASR articles

tfosmark committed:
Merge pull request #27999 from jimdial/patch-35

Updates.

cephalin committed:
MVC18 bugs

tfosmark committed:
Merge pull request #27986 from gsacavdm/patch-3

Fixing typo

rjagiewich committed:
Merge pull request #27988 from JYOTHIRMAISURI/master

few updates to itsmc articles

Commit:
fix typos and formatting

Lisaco88 committed:
Merge pull request #27259 from alkohli/SSregion

StorSimple region availability

mmacy committed:
[aci] date correct

cephalin committed:
more bugs

tynevi committed:
Merge pull request #28008 from MicrosoftDocs/release-reserve-instance

Release reserve instance

rjagiewich committed:
Merge pull request #27539 from barclayn/keyvault-remove-portal

working on removing references to the classic portal

sethmanheim committed:
Update per customer comments

sethmanheim committed:
Acrolinx

rjagiewich committed:
Merge pull request #27832 from dksimpson/dean-656

task 656: Rewrite text explaining query string structure

rjagiewich committed:
Merge pull request #27960 from MarkMcGeeAtAquent/tsi-41

small Time Series Insights fixes

rjagiewich committed:
Merge pull request #28006 from mmacy/aci-article-date-fix

[aci] date correct

sethmanheim committed:
Tweak

Commit:
network perf to bandwidth

rjagiewich committed:
Merge pull request #28005 from jluk/master

fix typos and formatting

ktoliver committed:
Merge pull request #26359 from tdykstra/cscwk1

Add precompiled C# to event hubs, service bus, timer

ShawnJackson committed:
Update active-directory-passwords-faq.md

LizCasey committed:
Changes that align with directive in task 1077555: Update references of cores --> vCPU.

Commit:
another small tweak

v-thepet committed:
add lf to json

elioda committed:
IoT Edge: gw fixes

elioda committed:
IoT Edge: gw article date fix

Commit:
link

ShawnJackson committed:
Update active-directory-passwords-best-practices.md

rmca14 committed:
Merge pull request #26942 from tdykstra/languages

clarify languages status

v-thepet committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into newfix

ShawnJackson committed:
Update active-directory-passwords-customize.md

ktoliver committed:
Merge pull request #28010 from sethmanheim/eh-fix

Update per customer comments

v-thepet committed:
fix new broken link since first commit

ShawnJackson committed:
Update active-directory-passwords-data.md

MarkMcGeeAtAquent committed:
Merge pull request #11 from MicrosoftDocs/master

merge

ShawnJackson committed:
Merge pull request #27477 from LoriMerrick/two-active-directory-passwords

edit pass: two-active-directory-passwords

Commit:
Updated for 11-16 announcements

All region availability for service tags and augmented security rules.

ktoliver committed:
Merge pull request #28004 from cephalin/usability

MVC18 bugs

ShawnJackson committed:
Update load-balancer-ha-ports-overview.md

rmca14 committed:
Merge pull request #28003 from jimdial/deprecate-cliv1-arm

Deprecated RM CLI V1

ktoliver committed:
Merge pull request #26623 from JennieHubbard/patch-96

fixed link to old portal

ktoliver committed:
Merge pull request #28014 from v-thepet/newfix

add lf to json

Commit:
First para

jodebrui committed:
adding queries to identify all In-Memory OLTP objects

ShawnJackson committed:
Merge pull request #27478 from LoriMerrick/three-active-directory-passwords

edit pass: three-active-directory-passwords

ktoliver committed:
Merge pull request #27923 from MicrosoftGuyJFlo/SSPRWritebackStrings

Sspr writeback strings

Commit:
Added preview note to some tags.

spelluru committed:
fix spelling

ShawnJackson committed:
Merge pull request #26672 from Ja-Dunn/load-balancer-ha-ports-overview

edit pass: load-balancer-ha-ports-overview

MarkMcGeeAtAquent committed:
added blue Next Steps button

Commit:
Added SE link.

ranvijaykumar committed:
Making the product name consistent with other documents

Commit:
Deprecated CLIV1 multiple IP article.

ShawnJackson committed:
Update active-directory-saas-sap-hana-cloud-platform-identity-authentication-tutorial.md

ktoliver committed:
Merge pull request #28015 from fsautomata/master

IoT Edge: gateway article fixes

MarkMcGeeAtAquent committed:
change IoT hub -> IoT Hub

MarkMcGeeAtAquent committed:
format fix?

ktoliver committed:
Merge pull request #28021 from JennieHubbard/patch-99

First para

ShawnJackson committed:
Merge pull request #26635 from rjagiewich/active-directory-saas-sap-hana-cloud-platform-identity-authentication-tutorial

Edit pass: Active directory saas sap hana cloud platform identity authentication tutorial

ktoliver committed:
Merge pull request #28028 from cychua/master

Update sync req on groups

spelluru committed:
more

raiye committed:
Update cloud-services-guestos-update-matrix.md

MarkMcGeeAtAquent committed:
format fix?

Commit:
changes for v6.0CU2

CarlRabeler committed:
Update sql-database-recovery-using-backups.md

Commit:
Updates.

LizCasey committed:
Extensive copy edits for a file not meeting minimum Acrolinx score.

ktoliver committed:
Merge pull request #28022 from jodebrui/patch-8

adding queries to identify all In-Memory OLTP objects

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

ktoliver committed:
Merge pull request #28027 from jimdial/deprecate-cliv1-multiple-ips

Deprecated CLIV1 multiple IP article.

MarkMcGeeAtAquent committed:
art fix

MarkMcGeeAtAquent committed:
last format fix try

Ja-Dunn committed:
Merge pull request #28031 from raiye/patch-1

Update cloud-services-guestos-update-matrix.md

Ja-Dunn committed:
Merge pull request #27970 from cynthn/net-sizes

Networking sizes fix

MatkovIvan committed:
Update ms.date

tdykstra committed:
correct bug introduced in PR 1873

Ja-Dunn committed:
Merge pull request #28034 from CarlRabeler/patch-364

Update sql-database-recovery-using-backups.md

MarkMcGeeAtAquent committed:
last format fix attempt

MarkMcGeeAtAquent committed:
added blue next steps button

eamonoreilly committed:
Added fix to Select content

Ja-Dunn committed:
Merge pull request #28017 from LizCasey/v-elicas1114

Changes that align with directive in task 1077555: Update references …

v-shils committed:
Merge pull request #28033 from v-payc/changesforv6.0CU2

Changesforv6.0 cu2

v-shils committed:
Merge pull request #28030 from spelluru/typo1116

fix spelling

v-shils committed:
Merge pull request #28025 from ranvijaykumar/patch-1

Making the product name consistent with other documents

v-shils committed:
Merge pull request #27885 from MatkovIvan/patch-2

Update version of the SDK

v-shils committed:
Merge pull request #28037 from tdykstra/exampleintro

correct bug introduced in PR 1873

Ja-Dunn committed:
Merge pull request #27884 from eamonoreilly/master

Adding watcher task tutorial to Automation documentation

divyaswarnkar committed:
c1

MarkMcGeeAtAquent committed:
added blue next steps

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

Ja-Dunn committed:
Merge pull request #28019 from jimdial/patch-36

Updated for 11-16 announcements

nitinme committed:
minor changes to incorporate feedback

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

divyaswarnkar committed:
c2

alkohli committed:
Fixed a broken link

alkohli committed:
Update storsimple-regions.md

Ja-Dunn committed:
Merge pull request #28024 from MarkMcGeeAtAquent/tsi-50

Updates to TSI topics

Ja-Dunn committed:
Merge pull request #28043 from alkohli/patch-157

Fixed a broken link

AnithaAdusumilli committed:
Update region availability for SQL

tfitzmac committed:
added multiple arrays

Lisaco88 committed:
Merge pull request #28041 from divyaswarnkar/soapupdate

c1

Lisaco88 committed:
Merge pull request #28042 from nitinme/nitinme-databricks-updates

Minor changes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #28053 from tfitzmac/1116variables

added multiple arrays

Lisaco88 committed:
Merge pull request #28052 from AnithaAdusumilli/patch-4

Update region availability for SQL

v-alje committed:
Merge pull request #28062 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
